src/Serilog.Sinks.Loggly/Sinks/Loggly/LogglySink.cs

```diff
@@ -80,7 +81,24 @@ public LogglySink(IFormatProvider formatProvider, int batchSizeLimit, TimeSpan p
         /// not both.</remarks>
         protected override async Task EmitBatchAsync(IEnumerable<LogEvent> events)
         {
-            await _client.Log(events.Select(_converter.CreateLogglyEvent));
+            try
+            {
+                LogResponse response = await _client.Log(events.Select(_converter.CreateLogglyEvent)).ConfigureAwait(false);
+
+                switch (response.Code)
+                {
+                    case ResponseCode.Error:
+                        SelfLog.WriteLine("LogglySink received an Error response: {0}", response.Message);
+                        break;
+                    case ResponseCode.Unknown:
+                        SelfLog.WriteLine("LogglySink received an Unknown response: {0}", response.Message);
+                        break;
+                }
+            }
+            catch (Exception ex)
```

Comment:
The exception shouldn't be caught here, but rather propagate up to Serilog (just removing the try/catch block here is fine).

Reply:
Done.  Thanks for taking a look!